internal/api/schemas/v1.yaml

```
@@ -11,17 +11,168 @@ info:
     name: Mozilla Public License 2.0
     url: https://www.mozilla.org/en-US/MPL/2.0/
 paths:
+  "/namespaces/{namespace}/gateways":
+    get:
+      description: 'Returns all configured gateways in the given namespace.'
+      operationId: listNamespacedGateways
```

"namespaced" describes every gateway IMO, so I'd go with `listGatewaysInNamespace` here

```suggestion
      operationId: listGatewaysInNamespace
```